video/profiles.go

```diff
@@ -134,7 +142,10 @@ func GetDefaultPlaybackProfiles(video InputTrack) ([]EncodedProfile, error) {
 		// check it here.
 		lowerQualityThanSrc := profile.Height < video.Height && profile.Bitrate < video.Bitrate
 		if lowerQualityThanSrc {
```

Comment:
Don't we also want this for the rendition that's the same resolution as the source?

Reply:
Ohh wait, maybe you're right that we should also increase the bitrate in line 157. Will update it there.

Reply:
Updated in 13a6bf2ed7c79f1f2e5781d5f46123dd4c16dced